.gitignore

```diff
@@ -0,0 +1,52 @@
+
+# Created by https://www.gitignore.io/api/ansible,linux,vim,virtualenv,git
```

Comment:
Can i disable protected for master and re-enable it later without interrupting the pull request?